### PR TITLE
Updated to latest versions

### PR DESCRIPTION
**Pull request by moo2u2**
Updated to:
- .NET Standard 2.1
- .NET Core 3.1 
- latest NuGet packages

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+This is a fork of the original at https://github.com/jss-blazor/jss-blazor updated to .NET Standard 2.1 and .NET Core 3.1
```
Comment (coreyasmith) [inline src/README.md]: Please revert these changes to the `README.md`.

File: samples/Styleguide/src/Project/Styleguide/client/Components/Styleguide-Layout.razor [hunk 2]
```diff
@@ -10,18 +10,18 @@
         <Placeholder Name="jss-styleguide-layout" Rendering="@Component" />
     </div>
     <div class="col-sm-4 col-lg-2 order-sm-first pt-2">
-        @foreach (var section in _sections)
+        @foreach (var s in _sections)
```
Comment (coreyasmith) [inline samples/Styleguide/src/Project/Styleguide/client/Components/Styleguide-Layout.razor:13]: Please revert these variable name changes.

File: src/JssBlazor.Components/Extensions/ComponentsApplicationBuilderExtensions.cs [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 using Toolbelt.Blazor.Extensions.DependencyInjection;
 
 namespace JssBlazor.Components.Extensions
 {
     public static class ComponentsApplicationBuilderExtensions
     {
-        public static void UseJssBlazorComponents<TComponent>(
-            this IComponentsApplicationBuilder app,
-            string domElementSelector)
-            where TComponent : IComponent
```
Comment (coreyasmith) [inline src/JssBlazor.Components/Extensions/ComponentsApplicationBuilderExtensions.cs:12]: The DOM element selector shouldn't go away--users should be able to specify their own DOM element selector for their app. Has this functionality been removed from Blazor?

**Review by coreyasmith — CHANGES_REQUESTED** [review]
I'll take a look at the updates to Blazor and give this pull request a more thorough review if I have time this weekend. Thanks for the pull request! 💃

File: src/README.md (hunk 1 above)
Comment (moo2u2) [reply src/README.md]: Sorry didn't mean to include this!

File: samples/Styleguide/src/Project/Styleguide/client/Components/Styleguide-Layout.razor (hunk 2 above)
Comment (moo2u2) [reply samples/Styleguide/src/Project/Styleguide/client/Components/Styleguide-Layout.razor:13]: Might just be my environment, but my VS2019 is interpreting line 15 as [razor @section directives](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout?view=aspnetcore-3.1#sections) and I get the errors: The 'section` directive must appear at the start of the line; The 'section' directives value(s) must be separated by whitespace.

File: src/JssBlazor.Components/Extensions/ComponentsApplicationBuilderExtensions.cs (hunk 3 above)
Comment (moo2u2) [reply src/JssBlazor.Components/Extensions/ComponentsApplicationBuilderExtensions.cs:12]: It hasn't been removed, just moved to the WebAssemblyHostBuilder.  I have separated the functionality into 2 methods (including the version with DOM element selector) and renamed the file to something a bit more generic.  Let me know if you prefer different naming for the methods/file.
